guide/14-deep-learning/finetune-sam-using-samlora.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<p>Are these strings supposed to exist like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1801/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Are you talking about the image encoder string. It renders as an image in the notebook.</p>